.github/workflows/community.yml

```diff
@@ -17,17 +17,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: ./src/welcome
+      - uses: EddieHubCommunity/gh-action-community/src/welcome@main
```

Comment:
Shouldn't we be using the latest release not main?

Reply:
Release would be better 👍 , but I think it was using `main` before. This can be updated here or in another PR